packages/ui/src/multiplying-architecture/KaotoEditorChannelApi.ts

```diff
@@ -57,4 +57,31 @@ export interface KaotoEditorChannelApi extends KogitoEditorChannelApi {
     exclude?: string,
     options?: Record<string, unknown>,
   ): Promise<string[] | string | undefined>;
+
+  /**
+   * Return the runtime information if the opened file is part of a Maven project.
+   * The returned Object is a direct parsing of the json string coming from Camel JBang.
+   * In case it is not a Maven project or a problem occured, undefined is returned.
+   */
+  getRuntimeInfoFromMavenContext(): Promise<
```

Comment:
Use this type instead: https://github.com/KaotoIO/kaoto/pull/2367/files#diff-c17789df4f7d321352506e04033bdb0614418d893fde485eb2d872ac0648cda8R1